src/scripts/pokemons/PokemonList.ts

```diff
@@ -16379,7 +16380,7 @@ const pokemonList = createPokemonArray(
         'id': 746.02,
         'name': 'Totem Wishiwashi (School)',
         'type': [PokemonType.Water],
-        'eggCycles': 15,
+        'eggCycles': 23,
```

Comment:
Does this actually affect it's egg steps?

Reply:
Yes it's 920 up from 600.
![WishiwashiSteps](https://user-images.githubusercontent.com/69112975/154836957-28b90f54-9aab-4017-b35e-f44404a9866e.png)

Reply:
Ah sorry, didn't realise this was for the totem variant 👍